extension/resources/templates/report_class.liquid

```diff
@@ -0,0 +1,86 @@
+<head>
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src *; style-src 'nonce-self'; script-src 'nonce-check';">
+    <style nonce="self">

```

Comment:
You can create a bootstrap stylesheet and reference it in a minute. Goto getbootstrap.com,